examples/errors.rs

```diff
@@ -0,0 +1,103 @@
+//! A Wasm module can sometimes not work, it will produce errors in some
+//! cases.
```

Comment:
//! A Wasm module can sometimes be invalid or trigger traps, and in those case we will get
//! an error back from the API.